server/migrations/20201223183016-add-sharing-permissions-associations.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, DataTypes) => {
+    return queryInterface.createTable('SharingPermissions', {
```

Comment:
Can we call this Collaborations instead of introducing a new term?

Reply:
Sure! That seems like a good call.